flow-server/src/main/resources/com/vaadin/flow/server/frontend/dependencies/react/package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "flow-default-react-dependencies",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@polymer/polymer": "3.5.1",
+    "@vaadin/common-frontend": "0.0.18",
+    "construct-style-sheets-polyfill": "3.1.0",
+    "lit": "3.0.2",
```

``
```suggestion
    "lit": "3.1.0",
```

Artur has updated it to 3.1.0 --> updated that no mismatch happens

---

Need to have a shared/common dependencies so the versions do not differ.